client/src/components/ErrorMessageComponent/ErrorMessage.js

```diff
@@ -59,6 +64,13 @@ export class ErrorMessageComponent extends Component {
     this.setState({ showModal: false });
   }
 
+  checkUrl = async () => {
+    const { appAdtUrl } = await configService.getConfig();
+    const urlValues = appAdtUrl.split(".");
+    const chinaUrl = urlValues[urlValues.length - 1] === "cn";
```

Comment:
Can we add a check here for the length of the urlValues array? I can't imagine this is a common scenario but just to be on the safe side. I.e. `urlValues.length > 0 && urlValues[urlValues.length - 1] === "cn";`.

Reply:
Ready